contracts/storage/ParametersStorage.sol

```diff
@@ -24,6 +25,8 @@ contract ParametersStorage is Ownable {
     uint24 public rewardWithdrawalDelay;
     uint32 public slashingFreezeDuration;
 
```

Comment:
We should add a mapping here named "customParameter" that we can populate if it happens to be that we need addition in the future (redeploying this contract should also not be a problem, lets consider the implications)